sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/aggregation/MinAggregator.java

```diff
@@ -15,6 +16,30 @@ public MinAggregator() {
 
     @Override
     public void aggregate(Object item) {
+
+        if (item instanceof ObjectNode) {
+            ObjectNode objectNode = (ObjectNode) item;
+            if (objectNode.hasNonNull("count")) {
+                long count = objectNode.get("count").asLong();
+                if (count == 0) {
+                    // Ignore the value since the continuation / partition had no results that matched the filter
+                    // so min/max is undefined.
+                    return;
+                }
+
+                if (objectNode.has("min")) {
```

Comment:
we should make this constants, ditto. here and elsewhere

Reply:
Done